mkstage4.sh

```diff
@@ -12,23 +12,23 @@ EXCLUDE_BOOT=0
 EXCLUDE_CONNMAN=0
 EXCLUDE_LOST=0
 QUIET=0
-USER_EXCL=""
+USER_EXCL=()
```

Comment:
I don't understand exactly what `()` does. Could you explain?

Reply:
() means to make an empty array.